features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -926,6 +926,10 @@
             <RecoveryLink>{{identity_mgt.password_reset_by_admin.enable_emailed_link_based_reset}}</RecoveryLink>
         </AdminPasswordReset>
         <CallbackRegex>{{identity_mgt.recovery.callback_url}}</CallbackRegex>
+        <ChallengeQuestion>
```

Comment:
`<Question>` tag is also for the challenge questions right? Why do we need a new tag?

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/2919/commits/ea7af90737d21f62569a67c50cfdc58d7f96e025